examples/openfin/app.json

```diff
@@ -4,7 +4,7 @@
         "name": "desktopJS Demo",
         "description": "desktopJS Demo",
         "url": "http://localhost:8000/index.html",
-        "uuid": "0td3jltz4vswz5mi",
+        "uuid": "desktopJS",
```

Comment:
OpenFin uses uuid as the name of the main window.  To provide for a simple consistent example for window group joining to the main window, change the uuid to "desktopJS" and use matching name in electron.js bootstrap.